### PR TITLE
Create staicase_pattern.py

### PR DESCRIPTION
**Pull request by Adityanagraj**
added a staircase pattern program in python under hakin codes open source contribution

### CODE REVIEW

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Follow [PEP8 Guidelines ](https://www.python.org/dev/peps/pep-0008/) carefully. 
Document your code. Include main function & using meaningful variable names.

Check File Name spelling.

**Comment by Adityanagraj** [comment]
I have updated the program with main function and variables @SKAUL05

File: Basic-Scripts/staicase.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""
+Which prints the staircase pattern
+   #
+  ##
+ ###
+####
+
+"""
+def staircase(height):
+   for i in range(1,height+1):
```
Comment (SKAUL05) [inline Basic-Scripts/staicase.py]: Use meaningful names

File: Basic-Scripts/staicase.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+"""
+Which prints the staircase pattern
+   #
+  ##
+ ###
+####
+
+"""
+def staircase(height):
```
Comment (SKAUL05) [inline Basic-Scripts/staicase.py]: Add function documentation

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Change filename to staircase_pattern

File: Basic-Scripts/staicase_pattern.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ ###
+####
+
+"""
+def staircase(height):
+   """
+   This program prints the staircase pattern for the given height (dynamically)
+   """
+   for i in range(1,height+1):
+       print(' '*(height-i) + '#'*i)
```
Comment (SKAUL05) [inline Basic-Scripts/staicase_pattern.py]:
```suggestion
   for index in range(1,height+1):
       print(' '*(height-index) + '#'*index)
```

File: Basic-Scripts/staicase_pattern.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      
+      
+if __name__ == '__main__':
+
+   print('Please enter the height of the staircase')      
+   height=int(input()) #which accepts the input
```
Comment (SKAUL05) [inline Basic-Scripts/staicase_pattern.py]:
```suggestion
   height=int(input('Please enter the height of the staircase')) #which accepts the input
```

File: Basic-Scripts/staicase_pattern.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Which prints the staircase pattern
+   #
+  ##
+ ###
+####
+
+"""
+def staircase(height):
+   """
+   This program prints the staircase pattern for the given height (dynamically)
```
Comment (SKAUL05) [inline Basic-Scripts/staicase_pattern.py:11]:
```suggestion
   This program prints the staircase pattern for the given height (dynamically)
   Params:
       height: height of the staircase as entered by user
   Returns:
       None
```

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Comment by SKAUL05** [comment]
Change filename to `staircase_pattern.py`

**Comment by Adityanagraj** [comment]
what changes i have to do

**Comment by SKAUL05** [comment]
> what changes i have to do

Changes suggested above

**Comment by Adityanagraj** [comment]
I have updated the file from staircase.py to staircase_pattern.py and changed the the code what else i need to do @SKAUL05

**Review by SKAUL05 — APPROVED** [review]
Good to go

**Comment by Adityanagraj** [comment]
when will the PR gets merged @SKAUL05

**Comment by SKAUL05** [comment]
@ankitdobhal Merge pls

**Pull request merged**